doc/start/quick-ceph-deploy.rst

```diff
@@ -25,181 +25,137 @@ are in this directory when executing ``ceph-deploy``.
    if you are logged in as a different user, because it will not issue ``sudo``
    commands needed on the remote host.
 
-.. topic:: Disable ``requiretty``
```

Comment:
Just that there were a bunch of other similar pitfalls (e.g., passwordless sudo, firewalld, etc) and it doesn't make sense to duplicate them all here.  Seems better to keep the directions as concise as possible?